addons/l10n_nz/data/account_tax_template_data.xml

```diff
@@ -280,24 +280,28 @@
             (0,0, {
                 'factor_percent': 100,
                 'repartition_type': 'base',
+                'plus_report_line_ids': [ref('tax_report_box11')],
```

Comment:
Where does the value come from?
From what I read on the opw:
> What would be your expected behavior in this case?
> That the GST on import shows in the Tax Report under Purchases and expenses and most likely in BOX 13 and BOX 14

Reply:
From what I understood, it seems more logic in BOX11
https://github.com/odoo/odoo/blob/146bf63b58b360e3453fee90585866ec637341b7/addons/l10n_nz/data/account_tax_report_data.xml#L76-L100

Reply:
I'm really not sure about that, and I would like to have the advice of a functional expert